synapse/lib/stormlib/json.py

```diff
@@ -45,3 +121,14 @@ async def _jsonLoad(self, text):
         except Exception as e:
             mesg = f'Text is not valid JSON: {text}'
             raise s_exc.BadJsonText(mesg=mesg)
+
+    async def _jsonSchema(self, schema):
+        schema = await s_stormtypes.toprim(schema)
+        # We have to ensure that we have a valid schema for making the object.
+        try:
+            await s_coro.forked(compileJsSchema, schema)
+        except asyncio.CancelledError:  # pragma: no cover
+            raise
+        except Exception as e:
+            raise s_exc.StormRuntimeError(mesg='Unable to compile Json Schema', schema=schema) from e
```

Comment:
We should try to include any additional context provided by the exception instead of just saying it's broke :)

Reply:
I've added the `str` of the exception to the message. Unfortunately that's about as far as we can go, since we have no specific exceptions that we could catch during the schema validations.